letter-sender/src/main/java/com/christmas/letter/sender/SampleController.java

```diff
@@ -1,16 +1,30 @@
 package com.christmas.letter.sender;
 
-import org.springframework.web.bind.annotation.GetMapping;
+import com.christmas.letter.sender.model.Letter;
+import com.christmas.letter.sender.service.SnsService;
+import io.micrometer.common.lang.NonNull;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
-@RequestMapping("/sample")
+@RequestMapping("/api/letters")
+@RequiredArgsConstructor
 public class SampleController {
```

Comment:
Let's rename this to something more friendly, maybe LetterController.

Reply:
changed to LetterController